line-sdk/src/main/java/com/linecorp/linesdk/internal/nwclient/TalkApiClient.java

```diff
@@ -56,21 +63,31 @@
     /* package */ static final String PATH_OTT_SHARE = "ott/share";
     /* package */ static final String PATH_OTT_ISSUE = "ott/issue";
 
+    @NonNull
+    private final Uri apiBaseUrl;
+
+    @NonNull
+    private final ChannelServiceHttpClient httpClient;
+
     private static final ResponseDataParser<LineProfile> PROFILE_PARSER = new ProfileParser();
-    private static final ResponseDataParser<LineFriendshipStatus> FRIENDSHIP_STATUS_PARSER =
-            new FriendshipStatusParser();
+    private static final ResponseDataParser<LineFriendshipStatus> FRIENDSHIP_STATUS_PARSER = new FriendshipStatusParser();
     private static final ResponseDataParser<GetFriendsResponse> FRIENDS_PARSER = new FriendsParser();
     private static final ResponseDataParser<GetGroupsResponse> GROUP_PARSER = new GroupParser();
+    private static final ResponseDataParser<List<SendMessageResponse>> MULTI_SEND_RESPONSE_PARSER = new MultiSendResponseParser();
+    private static final ResponseDataParser<OpenChatRoomInfo> OPEN_CHAT_ROOM_INFO_PARSER = new OpenChatRoomInfoParser();
+    private static final ResponseDataParser<OpenChatRoomStatus> OPEN_CHAT_ROOM_STATUS_PARSER = new OpenChatRoomStatusParser();
+    private static final ResponseDataParser<MembershipStatus> OPEN_CHAT_MEMBERSHIP_PARSER = new MembershipStatusParser();
 
     public TalkApiClient(Context applicationContext, @NonNull Uri apiBaseUrl) {
         this(apiBaseUrl, new ChannelServiceHttpClient(applicationContext, BuildConfig.VERSION_NAME));
     }
 
     @VisibleForTesting
-    protected TalkApiClient(
+    public TalkApiClient(
```

Comment:
This constructor is only used for testing, I think we don't need to expose it in public.